keyboards/doio/kb16/rev1/rules.mk

```diff
@@ -0,0 +1,40 @@
+SRC += lib/layer_status/layer_status_1.c
+SRC += lib/layer_status/layer_status_2.c
+SRC += lib/layer_status/layer_status_3.c
+SRC += lib/layer_status/layer_status_4.c
```

Comment:
Might be simpler to just make this an array of arrays, so you have one variable. 

And then you could run `oled_write_raw_P(layer_status[get_highest_layer(layer_state)], sizeof(layer_status[0])`